scenarios/kubernetes_e2e.py

```diff
@@ -476,7 +481,21 @@ def main(args):
         runner_args.append(
             '--gcp-service-account=%s' % mode.add_service_account(args.service_account))
 
-    if args.build is not None:
+    if args.use_shared_build is not None:
+        # find shared build location from GCS
+        build_file = ''
+        if args.use_shared_build:
+            build_file += args.use_shared_build + '-'
+        build_file += 'build-location.txt'
+        gcs_path = os.path.join(args.gcs_shared, os.getenv('PULL_REFS', ''), build_file)
+        try:
+            # tell kubetest to extract from this location
+            args.kubetest_args.append('--extract=' + read_gcs_path(gcs_path))
```

Comment:
how about print `gcs_path` so that we can have more debug clue

Reply:
done